src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1128,8 +1128,12 @@ LedgerManagerImpl::processFeesSeqNums(
                 }
                 else
                 {
-                    throw std::runtime_error(
-                        "found unexpected MAX_SEQ_NUM_TO_APPLY");
+                    ltxe.currentGeneralized()
```

Comment:
I'm a bit skeptical about this. This would_definitely be a bug, and hiding bugs is scary because you don't know the root cause. Is it something mild or something bad? For example, imagine that there were a bug that left `MAX_SEQ_NUM_TO_APPLY` in `LedgerTxn` between ledgers. Then nodes which were started at different times wouldn't necessarily get the same result. It's probably better to throw.

Reply:
Yeah that's a valid point. I removed that fixup commit.